common/useFissionAuth.ts

```diff
@@ -0,0 +1,171 @@
+import { useEffect, useState } from 'react'
+
+import * as webnative from 'webnative'
+import * as webnativeFilecoin from 'webnative-filecoin';
+
+import * as C from '@common/constants';
+import * as R from '@common/requests';
+
+import Cookies from 'js-cookie';
+
+webnative.setup.debug({ enabled: true })
+
+export function useFissionAuth() {
+  const [state, setState] = useState<webnative.State>(null)
+  let fs;
+  let authScenario: webnative.Scenario | null = null;
+  let username: string = null;
+
+
+  /** Webnative Initialization
+   * Load webnative and configure permissions.
+   */
+
+  useEffect(() => {
+    async function getState() {
+      const result = await webnative.initialise({
+        permissions: {
+          // The Estuary token is stored in app storage
+          // at bgins/estuary-www.
+          app: {
+            name: 'estuary-www',
+            creator: 'bgins',
```

Comment:
is this going to be permanent? should we have a `arg` user?

Reply:
Not permanent at all! I've updated that to `arg` and reserved the user. We can get folks working on the project set up as `arg` at some point.